src/inverseRobot.js

```diff
@@ -25,6 +25,21 @@
  */
 function inverseRobot(robot) {
   // write code here
+  try {
```

Comment:
try/catch block here is useless, you shouldn't get any unexpected errors from this task.